<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectTemplate</class>
 <widget class="QWidget" name="ProjectTemplate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Template Management</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="ListsLabelName">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Project Template</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListView" name="listListView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="NewListBtn">
     <property name="text">
      <string>New from current project</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="updateBtn">
     <property name="text">
      <string>Update from current project</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="RenameBtn">
     <property name="text">
      <string>Rename</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="FinishBtn">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="RemoveBtn">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listListView</tabstop>
  <tabstop>NewListBtn</tabstop>
  <tabstop>updateBtn</tabstop>
  <tabstop>RenameBtn</tabstop>
  <tabstop>RemoveBtn</tabstop>
  <tabstop>FinishBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
